packages/legend-graph/src/models/DomainModifierHelper.ts

```diff
@@ -0,0 +1,373 @@
+/**
```

Comment:
I thought this file should be in `@finos/legend-studio` instead. My idea is that records all of graph modifier interactions that Studio could make. Also, we could change the naming scheme a bit, e.g. `class__deleteProperty`